mobile_rewards/src/server.rs

```diff
@@ -442,3 +440,28 @@ fn sign_txn(txn: &BlockchainTxnSubnetworkRewardsV1, keypair: &Keypair) -> Result
     txn.reward_server_signature = vec![];
     Ok(keypair.sign(&txn.encode_to_vec())?)
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn get_time_range_returns_24_hour_period() {
+        let last_timestamp = Utc.ymd(2022, 11, 1).and_hms(0, 0, 0).timestamp();
+        let now = Utc.ymd(2022, 11, 2).and_hms(0, 0, 0);
+        let (start, stop) = get_time_range(last_timestamp, now);
+
+        assert_eq!(start, Utc.ymd(2022, 11, 1).and_hms(0, 0, 0));
```

Comment:
If this date is meant to be the same as `last_timestamp`, I'd rather it be made explicit by naming the value and reusing, than duplicating.

Reply:
But last_timestamp is a timestamp not a `DateTime<Utc>>`

Reply:
Fixed 9c82ebf